src/screens/FeedsManager/SupportedChainsCard.tsx

```diff
@@ -158,11 +164,11 @@ const renderOracle = (
     | FeedsManager_ChainConfigFields['ocr1JobConfig'],
 ) => (
   <>
-    <Grid item xs={12} sm={1} md={5}>
+    <Grid item xs={12} md={5}>
```

Comment:
i had to readjust the column sizing as i found an issue when the next  job is flowing onto the previous job.

Note the OCR2 is flowing behind OCR1 instead of a new row

Before:
<img width="988" alt="Screenshot 2024-09-12 at 5 00 44 PM" src="https://github.com/user-attachments/assets/d7ac79a6-2dc1-4db6-b53a-2f836409bc7f">

After: 
<img width="1617" alt="Screenshot 2024-09-12 at 5 36 54 PM" src="https://github.com/user-attachments/assets/8ba64c29-6cbb-40b7-bd84-7c55bafbb833">